internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/ControllerClient.java

```diff
@@ -1588,13 +1588,18 @@ private <T extends ControllerResponse> T request(
       Optional<String> controllerUrl) {
```

Comment:
This isn't something to fix in this PR, but a bit of context.  We dislike optionals.  Every optional in the code base is a little bit of tech debt IMO.  Killing optionals in the code base is definitely on my Christmas list. So uh... if you were wondering what to get me....

Reply:
I see. Won't promise to kill all optionals. I will aim to clean up this optional since I added it :D